aiogram/fsm/storage/base.py

```diff
@@ -91,6 +91,8 @@ async def close(self) -> None:  # pragma: no cover
 
 
 class BaseEventIsolation(ABC):
+    _locks: Dict[Hashable, Any]
```

Comment:
Why this attribute is added to the base class?
This attribute is needed only for in-memory events isolation